App.js

```diff
@@ -1,26 +1,191 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import styled from 'styled-components/native'
+import { ScrollView } from 'react-native'
+
+const App = () => {
+  const [isRunning, setIsRunning] = useState(false);
+  const [hundredths, setHundredths] = useState(0);
+  const [seconds, setSeconds] = useState(0);
+  const [minutes, setMinutes] = useState(0);
+  const [formattedTime, setFormattedTime] = useState('00:00,00');
+  const [laps, setLaps] = useState([]);
+
+  useEffect(() => {
+    if (isRunning) {
+      const interval = setInterval(() => {
+        setHundredths((hundredths) => {
+          if (hundredths === 99) {
+            setHundredths(0);
+            setSeconds((seconds) => {
+              if (seconds === 59) {
+                setSeconds(0);
+                setMinutes((minutes) => minutes + 1);
+              }
+              return seconds + 1;
+            });
+          }
+          return hundredths + 1;
+        });
+      }, 10);
+      return () => clearInterval(interval);
+    }
+  }, [isRunning]);
+
+  useEffect(() => {
+    const min = minutes <= 9 ? `0${minutes}` : minutes;
+    const sec = seconds <= 9 ? `0${seconds}` : seconds;
+    const hun = hundredths <= 9 ? `0${hundredths}` : hundredths;
+    setFormattedTime(`${min}:${sec},${hun}`);
+  }, [minutes, seconds, hundredths]);
+
+  const handleStart = () => {
+    setIsRunning(true);
+  };
+
+  const handleStop = () => {
+    setIsRunning(false);
+  };
+
+  const handleLap = () => {
+    const lap = {
+      num: laps.length + 1,
+      time: formattedTime
+    };
+    setLaps((laps) => [lap, ...laps]);
+  };
+
+  const handleReset = () => {
+    setHundredths(0);
+    setSeconds(0);
+    setMinutes(0);
+    setLaps([]);
+  };
+
+  return (
+    <Container>
+      <FixedSection>
+        <TimerText>{formattedTime}</TimerText>
+        <ButtonContainer>
+          <StartStopContainer>
+            {!isRunning ?
+              <StartButton onPress={() => handleStart()}>
+                <ButtonText>Start</ButtonText>
+              </StartButton>
+            :
+              <StopButton onPress={() => handleStop()}>
+                <ButtonText>Stop</ButtonText>
+              </StopButton>
+            }
+          </StartStopContainer>
+          <LapResetContainer>
+            {!isRunning ?
+              <ResetButton onPress={() => handleReset()}>
+                <ButtonText>Reset</ButtonText>
+              </ResetButton>
+            :
+              <LapButton onPress={() => handleLap()}>
+                <ButtonText>Lap</ButtonText>
+              </LapButton>
+            }
+          </LapResetContainer>
+        </ButtonContainer>
+      </FixedSection>
+      <ScrollView>
+        <LapContainer>
+          {laps.map((item) => (
+            <LapListItem 
+              key={item.num}>
```

Comment:
No need for line break here, you only pass one prop to `<LapListItem />` component